library/src/main/scala/scala/scalajs/runtime/WasPublicBeforeTyper.scala

```diff
@@ -0,0 +1,3 @@
+package scala.scalajs.runtime
+
+class WasPublicBeforeTyper extends scala.annotation.Annotation
```

Comment:
I still do not understand why this is in `runtime`. If this is in `runtime`, so should `ExposedJSMember`.